packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -194,6 +194,11 @@ class Slider extends React.Component {
 
   jumpAnimationTimeoutId = -1;
 
+  constructor(props) {
+    super(props);
+    this.handleMouseMoveBind = this.handleMouseMove.bind(this);
+    this.handleMouseUpBind = this.handleMouseUp.bind(this);
```

Comment:
We stick to lambdas as class instance properties.

Reply:
Ok. Will change it.